handlers.go

```diff
@@ -402,4 +402,8 @@ func forwardEvent(falcopayload types.FalcoPayload) {
 	if config.Dynatrace.APIToken != "" && config.Dynatrace.APIUrl != "" && (falcopayload.Priority >= types.Priority(config.Dynatrace.MinimumPriority) || falcopayload.Rule == testRule) {
 		go dynatraceClient.DynatracePost(falcopayload)
 	}
+
+	if config.OTLP.Traces.Endpoint != "" && (falcopayload.Priority >= types.Priority(config.OTLP.Traces.MinimumPriority)) {
+		go otlpClient.OTLPPost(falcopayload)
```

Comment:
I guess we may have more than traces in the future, so the function should be `OTLPTracesPost()`, wdyt?

Reply:
+1